src/taxonomy/taxonomy-menu/TaxonomyMenu.test.jsx

```diff
@@ -0,0 +1,250 @@
+import { useMemo } from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import PropTypes from 'prop-types';
+import each from 'jest-each';
```

Comment:
`jest-each` is included in jest, so you don't need to install the package or import is. Just do `describe.each` instead of `each.describe`